XenModel/InvisibleMessages.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="CLIENT_ID_URL" xml:space="preserve">
+    <value>http://support.citrix.com/xencenterclientiddownload</value>

```

Comment:
Link isn't working. Ignore if it's a placeholder/we're waiting for this to be online.